Doc/whatsnew/3.5.rst

```diff
@@ -2327,11 +2327,14 @@ The :func:`inspect.getargspec` function is deprecated and scheduled to be
 removed in Python 3.6.  (See :issue:`20438` for details.)
 
 The :mod:`inspect` :func:`~inspect.getfullargspec`,
-:func:`~inspect.getargvalues`, :func:`~inspect.getcallargs`,
-:func:`~inspect.getargvalues`, :func:`~inspect.formatargspec`, and
-:func:`~inspect.formatargvalues` functions are deprecated in favor of
-the :func:`inspect.signature` API.
-(Contributed by Yury Selivanov in :issue:`20438`.)
+:func:`~inspect.getcallargs`, and :func:`~inspect.formatargspec` functions are
+deprecated in favor of the :func:`inspect.signature` API. (Contributed by Yury
+Selivanov in :issue:`20438`.)
+
+.. note::
+
+  Function :func:`~inspect.getargvalues` and :func:`~inspect.formatargvalues`
```

Comment:
Or start with:

```rst
:func:`~inspect.getargvalues` and :func:`~inspect.formatargvalues` functions were inadvertently [...]
```

I'm not a native speaker so I hope @ncoghlan will chime in here :)